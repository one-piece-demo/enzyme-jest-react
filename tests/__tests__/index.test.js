const srcComponents = require('../../src/index');

describe('srcComponents index', () => {
  it('exports modules correctly', () => {
    expect(Object.keys(srcComponents)).toMatchSnapshot();
  });
});
