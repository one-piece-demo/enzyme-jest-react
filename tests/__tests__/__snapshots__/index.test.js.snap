// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`srcComponents index exports modules correctly 1`] = `
Array [
  "Calculator",
]
`;
